src/components/TodoActions.tsx

```diff
@@ -0,0 +1,40 @@
+
+import CardActions from '@mui/material/CardActions';
+import Button from '@mui/material/Button';
+import Box from '@mui/material/Box';
```

Comment:
Instead of importing each component individually, you can use destructuring to import multiple components from the same package on a single line. For example:

import { CardActions, Button, Box, Typography, TextField } from '@mui/material';